Corona-Warn-App/src/main/res/layout/vaccination_certificate_card.xml

```diff
@@ -45,7 +45,7 @@
 
     <TextView
```

Comment:
I think this text is a big bigger than the other Views with the`subtitle` style because a bigger `textsize` is defined on this view which should be removed?